contracts/Cross-chain/OmnichainProposalSender.sol

```diff
@@ -135,13 +134,13 @@ contract OmnichainProposalSender is ReentrancyGuard, BaseOmnichainControllerSrc
         _ensureAllowed("execute(uint16,bytes,bytes)");
 
         require(validChainIds[remoteChainId_], "OmnichainProposalSender: Invalid chainId");
+        require(payload_.length != 0, "OmnichainProposalSender: Empty payload");
 
         bytes memory trustedRemote = trustedRemoteLookup[remoteChainId_];
         require(trustedRemote.length != 0, "OmnichainProposalSender: destination chain is not a trusted source");
 
         //  Same proposal should not contain more than one command
         uint256 pId = governanceBravoDelegate.proposalCount();
```

Comment:
I don't think this is valid in any case. For example:
1. VIP-123 is proposed
2. VIP-124 is proposed
3. VIP-125 is proposed
4. VIP-123 is executed, and it contains a remote command. At this moment `governanceBravoDelegate.proposalCount();` will return 125, not 123. If VIP-125 contains remote commands, wouldn't the remote proposal been overwritten in the Executor contract?

Reply:
[5f92a0d](https://github.com/VenusProtocol/governance-contracts/pull/54/commits/5f92a0da4bea4802931cb71e82a2e9a92f92024e)